andela/strikers-ah-frontend
# #164489736 Set Up React Project

#### What does this PR do?
This PR creates an empty **React App**
#### Description of Task to be completed?
- Initialize an empty react app using **create-react-app**.
- Setup eslint with Airbnb Configuration.
- Install Husky and lint-staged to help in adhering on same eslint rules.
#### How should this be manually tested?
-  After cloning the repository with https://github.com/andela/strikers-ah-frontend`, check out to this branch `ch-set-up-react-project-164489736`
- Run dependency installation with **npm install**
- start the app with **npm start**

#### What are the relevant pivotal tracker stories?
#164489736
#### Screenshots (if appropriate)
<img width="329" alt="Screen Shot 2019-05-15 at 18 33 28" src="https://user-images.githubusercontent.com/37753967/57792729-0363df80-7740-11e9-9dc6-083ef9089d37.png">
